uportal-war/src/main/java/org/apereo/portal/portlets/lookup/PersonLookupHelperImpl.java

```diff
@@ -427,9 +427,9 @@ protected IAuthorizationPrincipal getPrincipalForUser(final IPerson person) {
      * @param attributeNames
      * @return
      */
-    protected Set<String> getAvailableAttributes(
+    protected Set<String> getPermittedAttributes(
```

Comment:
@apetro -- I'm supportive of the general practice of using `@since` in this way, but I'm in doubt about this case.

This isn't a new method, but a name change for clarity.  (The old name does not suggest that permissions checking is performed, leaving the the caller wondering if it needs doing.)